docs/native/src/scenarios/operational_flows.rst

```diff
@@ -0,0 +1,67 @@
+..  _operational flows:
+
+Operational flows
+=================
+
+This section outlines the implementation requirements and operational flows for user scenarios outlined in :ref:`Browsing, opening and saving files<browse files>` section. 
+
+**Terminology**
+ * *User:* Self-explanatory. 
+ * *Office Client:* The |Office iOS| app, e.g. Excel/Word/PPT 
+ * *Office Identity Service:* The Office Identity service which is Office's service for handling user identity information. In OAuth flows, the Office Identity Service behaves as the External User Agent
+ * *Bootstrapper URL:* URL which the Storage Provider hosts to allow an Office Client app with a valid Oauth 2.0 access token to retrieve a WOPI access token
+ * *Token Endpoint URL:* URL which the Storage Provider hosts & uses in order to generate access or refresh tokens 
+ * *Services Manager:* Office-owned service which stores a Service Catalog, which has all of the available services for an app or endpoint
+ * *Storage provider:* Refers to a CSPP partner who has integrated their service into |Office iOS|
+
+ ..  note::
+	 * Any time during these flows that we try to hit the bootstrapper and have expired or invalid OAuth credentials we will ask the user to log in again.
+	 * Any time during these flows that we have a missing, invalid, or expired WOPI access token we will attempt to get a valid WOPI access_token by calling :ref:`GetNewAccessToken on the Bootstrapper <GetRootContainer (bootstrapper)>`.
```

Comment:
Also, note that when using `:ref:` (or any other similar constructs like `:term`), you don't need to specify a title unless you want it to be different than the title of the section you're referring to. In general you shouldn't specify your own title when referring to a section unless you really need to to make the language sound right.